docs/groovy-script/mods/pedestalcrafting/pedestal_crafting.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: "Pedestal Crafting"
 titleTemplate: "Pedestal Crafting | CleanroomMC"
-description: "Converts a variable number of input items into a single output item. The recipe can be customized with various particles."
+description: "Converts a number of 2+ inputs into a single output item. The recipe can be customized with various particles."
 source_code_link: "https://github.com/Ender-Development/PedestalCrafting-Patched/blob/master/src/main/java/me/axieum/mcmod/pedestalcrafting/compat/groovyscript/PedestalCrafting.java"
 ---
 
 # Pedestal Crafting (Pedestal Crafting)
 
 ## Description
 
-Converts a variable number of input items into a single output item. The recipe can be customized with various particles.
+Converts a number of 2+ inputs into a single output item. The recipe can be customized with various particles.
```

Comment:
given the change, i think this should specify that it takes a center item and one or more surrounding pedestal items